worlds/minecraft/__init__.py

```diff
@@ -11,11 +11,17 @@
 
 from BaseClasses import Region, Entrance, Item
 from .Options import minecraft_options
-from ..AutoWorld import World
+from ..AutoWorld import World, WebWorld
 
 client_version = 7
 minecraft_version = "1.17.1"
 
+
+class MinecraftWebWorld(WebWorld):
+    theme = "jungle"
+    bug_report_page = "https://github.com/KonoTyran/Minecraft_AP_Randomizer/issues"
```

Comment:
Not critical but you could make a bug report issue template and link to creating a new issue with that template directly. I believe this is possible with GitHub.